mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndex.java

```diff
@@ -99,6 +99,7 @@ public ListenerType getType() {
         Preconditions.checkArgument(session != null, "'session' is mandatory");
         return searcher.search(ImmutableList.of(session.getUser()), searchQuery, Optional.of(limit))
             .map(SearchResult::getMessageId)
+            .filter(com.google.common.base.Optional::isPresent)
```

Comment:
Please remove this line, or add log. We need to know when a multisearch result do not have MessageId